### PR TITLE
APPS-1249 fix missing required input cmd

### PR DESCRIPTION
**Pull request by YuxinShi0423**

### CODE REVIEW

File: src/python/dxpy/templating/templates/nextflow/src/nextflow.sh [hunk 1]
```diff
@@ -365,7 +367,7 @@ main() {
     $RESUME_CMD \
     $nextflow_run_opts \
     $nextflow_pipeline_params \
-    @@REQUIRED_RUNTIME_PARAMS@@
+    $required_inputs
```
Comment (YuxinShi0423) [inline src/python/dxpy/templating/templates/nextflow/src/nextflow.sh:370]: need to work together with
https://github.com/dnanexus/dx-toolkit/blob/master/src/python/dxpy/nextflow/nextflow_templates.py#L83-L87

**Review by commandlinegirl — APPROVED** [review]
LGTM

**Comment by commandlinegirl** [comment]
Per discussion: please update the commit ID (jira ticket) to the newest. Thanks!

**Comment by YuxinShi0423** [comment]
closed and fixed in https://github.com/dnanexus/dx-toolkit/pull/887

**Pull request closed without merging**